ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -202,6 +232,107 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         }
     }
 
+	private void handleAuthenticatedUser(JSONObject config, HttpServletRequest request, String xmppDomain, JSONObject conferences)
```

Comment:
If we do need this (see earlier comment), can we make this method return `conferences`, instead of passing an object (that appears to be always empty at this stage?) as an argument?